core/src/avm2/globals/flash/system/System.as

```diff
@@ -0,0 +1,7 @@
+package flash.system {
+    public final class System {
+        public static function gc(): String {
```

Comment:
Oops don't merge it please, wrong return type :)

Reply:
Quiet ping for this one-word fix. :no_mouth:

Reply:
never mind, sorry 😔

Reply:
fix'd